### DISCUSSION TITLE
The program should remember whether it was in full-screen or windowed mode the last time the user exited the program [Done]

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#2609] [Ideas]
When the user exits the program, the program should remember whether the game was in full-screen or windowed mode so that when the user restarts the program it is automatically in the same mode as when the user left it.
Basically, this should be updated in the game config file "fheroes2.cfg".

**Comment by ihhub** [comment]
Hi @LeHerosInconnu , please make it as an issue. This is definitely must have behavior.

**Reply by LeHerosInconnu** [reply]
Done.

**Comment by Branikolog** [comment]
Agreed!

**Comment by LeHerosInconnu** [comment]
This has been implemented.
I close the discussion.
